aries_vcx/src/common/ledger/transactions.rs

```diff
@@ -240,4 +281,43 @@ mod test {
                 .unwrap()
         );
     }
+
+    #[tokio::test]
+    async fn test_did_doc_from_oob_invitation_works() {
+        let mut did_doc = DidDoc::default();
+       // let recipient_keys = vec![_ed25519_public_key_to_did_key(_key_1().as_str()).unwrap()];
+        did_doc.set_id(MessageId::id().0);
+        did_doc.set_service_endpoint(_service_endpoint());
+        did_doc.set_recipient_keys(vec![_key_2()]);
+        did_doc.set_routing_keys(_routing_keys());
+        assert_eq!(
+            did_doc,
+            into_did_doc(&mock_profile(), &Invitation::OutOfBand(_oob_invitation()))
+                .await
+                .unwrap()
+        );
+    }
+
+
+    #[tokio::test]
+    async fn test_did_key_to_did_raw() {
+        // Test 1
+        let val1 = _key_1();
+        let val2 = _key_2();
+        let recipient_keys = vec![_ed25519_public_key_to_did_key(&val1).unwrap(),
```

Comment:
would be good to have some tests which use official ed25519 `did:key`s and verkey's from elsewhere, as testing against only your own implementation does not ensure the implementation is perfect.

e.g.  `did:key:z6MkwHgArrRJq3tTdhQZKVAa1sdFgSAs5P5N1C4RJcD11Ycv` should convert to `HqR8GcAsVWPzXCZrdvCjAn5Frru1fVq1KB9VULEz6KqY`

Reply:
or `did:key:z6MkukGVb3mRvTu1msArDKY9UwxeZFGjmwnCKtdQttr4Fk6i` which should convert to `_key_1()`.

Reply:
you could perhaps add these testing `did:key` strings alongside `_key_1` and `_key_2` in `messages/src/did_doc/mod.rs`